unit_tests/unit/nemesis/monkey/test_add_remove_dc.py

```
@@ -0,0 +1,239 @@
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *exc):
+        self.rollback_calls.append(self.preserved.copy())
+        self.events.append(f"{self.name}:rollback")
+        return False
+
+    def __call__(self, **keyspaces):
+        self.calls.append(keyspaces)
+        for keyspace, strategy in keyspaces.items():
```

Ruff (0.15.14)

[warning] Loop control variable `strategy` not used within loop body

Rename unused `strategy` to `_strategy`

(B007)